### PR TITLE
feat: update quantity of a cart item

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
Update the quantity of a cart item. API and consumable with tests.

## Related Issue
VBC-69

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/composables/__tests__/useCart/updateItemQty.spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { updateItemQty } from '../../src/useCart/actions';
+import { mockedCart } from '../../__mocks__/cart.mock';
+import { contextMock } from '../../__mocks__/context.mock';
+
+describe('[BigCommerce - composables] useCart updateItemQty', () => {
+  it('should add item to the cart', async () => {
```
Comment (scernisevs) [inline packages/composables/__tests__/useCart/updateItemQty.spec.ts:6]: Hi @zoltantakacs-born , should this be 'update cart item qty'? :)

**Review by scernisevs — APPROVED** [review]
LGTM :) thank you!

**Review by dhargitai — APPROVED** [review]
LGTM 👍

**Review by WojtekTheWebDev — APPROVED** [review]
LGTM

**Pull request merged**